content/pt/blog/2025/how-to-name-your-metrics.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Como Nomear Suas Métricas
+linkTitle: Como Nomear Suas Métricas
+date: 2025-09-10
+author: >-
+  [Juraci Paixão Kröhling](https://github.com/jpkrohling) (OllyGarden)
+canonical_url: https://blog.olly.garden/how-to-name-your-metrics
+default_lang_commit: b570deff7bac1978dd636ee724d3c00c18727c05
+# prettier-ignore
+cSpell:ignore: apiserver ecommerce jpkrohling kubelet mebibytes OllyGarden opentelemetrys postgres scheduler UCUM
+---
+
+As métricas são a espinha dorsal quantitativa da observabilidade — os números
+que nos dizem como nossos sistemas estão se comportando. Esta é a terceira
+publicação em nossa série sobre nomenclatura do OpenTelemetry, onde já
+exploramos [como nomear trechos](/blog/2025/how-to-name-your-spans/) e
+[como enriquecê-los com atributos significativos](/blog/2025/how-to-name-your-span-attributes/).
+Agora, vamos abordar a arte de nomear as medições que realmente importam.
+
+Ao contrário dos trechos _(spans)_ que contam histórias sobre o que aconteceu,
+as métricas nos falam sobre quantidades: quantos, quão rápido, quanto. Mas aqui
```

Comment:
"how many and how much" both translate to "quanto", despite one being about gauges (last value) and counters (up or up/down), being repetitive here. Perhaps: 

```suggestion
as métricas nos falam sobre quantidades: quais, quantos e quão rápido. Mas aqui
```